components/Extensions/src/Shadows/AttachedDropShadow.cs

```diff
@@ -31,7 +31,7 @@ public sealed class AttachedDropShadow : AttachedShadowBase
 #endif
 
     /// <inheritdoc/>
-    protected internal override bool SupportsOnSizeChangedEvent => true;
+    public override bool SupportsOnSizeChangedEvent => true;
```

Comment:
Curious for all these `protected internal` cases why `protected` isn't enough either... @ryken100 thoughts?

Reply:
For this `SupportsOnSizeChangedEvent`, it's referenced in https://github.com/CommunityToolkit/Windows/blob/0bab2c0359ab33b7ef9868496a2fe5175c5afe6b/components/Extensions/src/Shadows/AttachedShadowElementContext.cs#L121

Which doesn't derive `AttachedDropShadow`. It seems like `protected internal` was allowing `protected` _or_ `internal` access, losing `internal` and being `protected` means only derived classes can access it. Correct me if I'm wrong, @Ryken100!